src/asciidoc/pravidla_kapitoly/_slovnik_pojmu.adoc

```diff
@@ -180,4 +180,4 @@ a jako takové se to zaznamená do výsledků jako DNS. Závodník, který je oz
 
 [[rozcvicovaci-zona]]Rozcvičovací zóna:: znamená jakoukoliv část <<#soutezni-prostor,Soutěžního prostoru>> která je určená a vybavená pro potřeby přípravy závodníků
 
-[[z-klip]]Z-klip:: je situace, kdy je lano protažené skrz dva <<#jistici-bod,Jistící body>> mimo logické pořadí
\ No newline at end of file
+[[z-klip]]Z-klip:: je situace, kdy je lano protažené skrz dva <<#jistici-bod,Jistící body>> mimo logické pořadí
```

Comment:
co jsi tu menila? prijde mi to stejne.

Reply:
Tady nic :) Ze začátku jsem s tím bojovala, tak jsem asi vytvořila zbytečnou "vidličku". Jinak jsem zatím neměla čas se na to kouknout, máme účetní uzávěrku a ještě zápočet ve škole. Snad příští týden...